lib/ansible/modules/network/cnos/cnos_conditional_command.py

```diff
@@ -0,0 +1,184 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2017 Lenovo, Inc.
+#
+# This file is part of Ansible
+#
+# Ansible is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Ansible is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
+#
+# Module to send Conditional CLI commands to Lenovo Switches
+# Lenovo Networking
+#
+
+ANSIBLE_METADATA = {'status': ['preview'],
+                    'supported_by': 'community',
+                    'version': '1.0'}
+
+DOCUMENTATION = '''
+---
+module: cnos_conditional_command
+short_description: This module executes a single CNOS command with respect to conditions specified in the inventory.
+description:
+   - This module allows you to modify the running configuration of a switch. It provides a way to
+    execute a single CNOS command on a network device by evaluating the current running configuration
+    and executing the command only if the specific settings have not been already configured.
+    The CNOS command is passed as an argument of the method.
+    This module functions the same as the cnos_command module.
+    The only exception is that the following inventory variable can be specified
+    [“condition = <flag string>”]
+    When this inventory variable is specified as the variable of a task, the command is executed for
+    the network element that matches the flag string. Usually, commands are executed across a group
+    of network devices. When there is a requirement to skip the execution of the command on one or
+    more devices, it is recommended to use this module.
+    This module uses SSH to manage network device configuration.
+    For more information about this module from Lenovo and customizing it usage for your
+     use cases, please visit our [User Guide](http://systemx.lenovofiles.com/help/index.jsp?topic=%2Fcom.lenovo.switchmgt.ansible.doc%2Fcnos_conditional_command.html)
```

Comment:
CI failure due to PEP 8 issue:

```
2017-02-22 17:49:26 ERROR: PEP 8: lib/ansible/modules/network/cnos/cnos_conditional_command.py:47:161: E501 line too long (166 > 160 characters) (current)
```

The PEP 8 tests can be run locally with `make pep8`.